app/javascript/packs/input.js

```diff
@@ -0,0 +1,25 @@
+let i = 0;
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I'd recommend using the turbolinks events, rather than the Dom events

```suggestion
document.addEventListener('turbolinks:load', function () {
```